src/services/ui/src/pages/actions/IssueRai.tsx

```diff
@@ -1,51 +1,179 @@
+            <strong className="bold">
+              If you leave this page, you will lose your progress on this form.
+            </strong>
+          </p>
+        </ActionFormIntro>
+        {/* Package ID and type info */}
+        <PackageInfo item={item} />
+        {/* Attachments */}
+        <h3 className="font-bold text-2xl font-sans">Attachments</h3>
+        <AttachmentsSizeTypesDesc
+          faqLink={"/faq/#medicaid-spa-rai-attachments"}
+        />
+        {attachments.map(({ name, label, required }) => (
+          <FormField
+            key={String(name)}
+            control={form.control}
+            name={`attachments.${String(name)}` as Path<typeof schema>}
+            render={SlotAttachments({
+              label: (
+                <>
+                  {label}
+                  {required ? <RequiredIndicator /> : ""}
+                </>
+              ),
+              message: <FormMessage />,
+              className: "my-4",
+            })}
+          />
+        ))}
+        {/* Additional Info */}
+        <FormField
+          control={form.control}
+          name={"additionalInformation" as Path<typeof schema>}
+          render={SlotAdditionalInfo({
+            label: (
+              <p>
+                Add anything else that you would like to share with the State.
+              </p>
+            ),
+            description: "4,000 characters allowed",
+            className: "pt-6",
+            required: true,
+          })}
+        />
+        {/* Error banner */}
+        {Object.keys(form.formState.errors).length !== 0 && (
+          <Alert className="my-6" variant="destructive">
+            Input validation error(s)
+            <ul className="list-disc">
+              {Object.values(form.formState.errors).map(
+                (err, idx) =>
+                  err?.message && (
+                    <li className="ml-8 my-2" key={idx}>
+                      {err.message as string}
+                    </li>
+                  )
+              )}
+            </ul>
+          </Alert>
+        )}
+        {/* Pre-submit message banner */}
+        <Alert variant={"infoBlock"} className="my-2 w-full flex-row text-sm">
+          <Info />
+          <p className="ml-2">
+            Once you submit this form, a confirmation email is sent to you and
+            to the State.
+          </p>
+        </Alert>
+        {/* Buttons */}
+        <div className="flex gap-2 my-8">
+          <Button type="submit">Submit</Button>
+          <Button
+            type="button"
+            variant="outline"
+            onClick={() => {
+              setModalContent({
+                header: "Stop form submission?",
+                body: "All information you've entered on this form will be lost if you leave this page.",
+                acceptButtonText: "Yes, leave form",
+                cancelButtonText: "Return to form",
+              });
+              setModalOnAccept(() => acceptAction);
```

Comment:
I haven't tested yet, but this looks like it needs to be updated to:

```js
setModalOnAccept(acceptAction);
```

Reply:
This works as-intended. To properly test/confirm, change the `() => acceptAction` to `acceptAction` and try the Cancel button on this, or the other, forms.